zipline/utils/serialization_utils.py

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright 2015 Quantopian, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Label for the serialization version field in the state returned by
+# __getstate__.
+VERSION_LABEL = '_stateversion_'
+
+
+class SerializeableZiplineObject(object):
```

Comment:
Why do we need to have this class at all? The getstate method is a perfectly good utility func. I think it was basic_get_state before. There really not a single subclass that doesn't override those methods.